integrations/docker/images/chip-build-imx/Dockerfile

```diff
@@ -0,0 +1,21 @@
+ARG VERSION=latest
+FROM connectedhomeip/chip-build:${VERSION} as build
+RUN set -x \
+    && apt-get update \
+    && DEBIAN_FRONTEND=noninteractive apt-get install -fy --no-install-recommends \
+    wget=1.20.3-1ubuntu2 \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/ \
+    && : # last line
+WORKDIR /opt
+RUN set -x \
+    && wget --quiet -O fsl-l5.10.52-2.1.0-sdk.tar.gz https://www.nxp.com/lgfiles/IMM/fsl-l5.10.52-2.1.0-sdk.tar.gz \
+    && tar zxvf fsl-l5.10.52-2.1.0-sdk.tar.gz \
+    && ./fsl-imx-xwayland-glibc-x86_64-imx-image-core-cortexa53-crypto-imx8mmevk-toolchain-5.10-hardknott.sh -y \
+    && : # last line
+
+FROM connectedhomeip/chip-build:${VERSION}
+
+COPY --from=build /opt/fsl-imx-xwayland /opt/fsl-imx-xwayland
+
+ENV IMX_SDK_ROOT=/opt/fsl-imx-xwayland/5.10-hardknott/
```

Comment:
could we add this  copy  and env variable in the `chip-build-vscode` image?

Reasoning is that if we do that, developers in vscode would be able to build imx without extra hoops and also cloudbuild uses vscode image and could validate thins.

For vscode in `.devcontainer` build instrunctions, we may need to chown/chmod the imx sdk so that the user `vscode` has access to these files.

Reply:
Thank you for the comments.
I've added in latest pull request for adding this into chip-build-vscode Dockerfile and .devcontainer/Dockerfile.

But in my local build, I saw that the target file I'm copying from chip-build-imx image not exist. So could you please help to review whether the changes are correct?

Very appreciate for you.
2022-01-06T00:43:35.135Z] Removing intermediate container 3c4c7ec6b0ac
---> 6604889071ad
Step 34/34 : RUN chown -R $USERNAME:$USERNAME /opt/fsl-imx-xwayland/5.10-hardknott/
[2022-01-06T00:43:35.212Z] ---> Running in 1f3710ad2b0d
**[2022-01-06T00:43:35.605Z] chown: cannot access '/opt/fsl-imx-xwayland/5.10-hardknott/': No such file or directory
[2022-01-06T00:43:35.873Z] The command '/bin/sh -c chown -R $USERNAME:$USERNAME /opt/fsl-imx-xwayland/5.10-hardknott/' returned a non-zero code: 1**
[2022-01-06T00:43:35.878Z] Stop (1172302 ms): Run: docker build -f /home/haoran/matter/connectedhomeip/.devcontainer/Dockerfile -t vsc-connectedhomeip-c5a584ec93ecdaa5596f15f47fd1a5c8 --build-arg BUILD_VERSION=0.5.40 /home/haoran/matter/connectedhomeip/.devcontainer
[2022-01-06T00:43:35.885Z] Command failed: docker build -f /home/haoran/matter/connectedhomeip/.devcontainer/Dockerfile -t vsc-connectedhomeip-c5a584ec93ecdaa5596f15f47fd1a5c8 --build-arg BUILD_VERSION=0.5.40 /home/haoran/matter/connectedhomeip/.devcontainer

Reply:
Hi, I think I know why I get this log. It is because that the chip-build-imx images is not ready in Docker Hub so that it didn't copy successfully. But I saw that the chip-build-k32w and other docker images are ready in Docker Hub. So will the Matter community handle the Docker Hub images upload, rigth?
I think this conversation can be mark resolved.